src/generators.rs

```diff
@@ -2499,15 +2514,32 @@ pub fn complete_graph(
 ///  graph = rustworkx.generators.directed_complete_graph(5)
 ///  mpl_draw(graph)
 ///
-#[pyfunction(multigraph = true)]
+#[pyfunction(multigraph = "true")]
 #[pyo3(text_signature = "(/, num_nodes=None, weights=None, multigraph=True)")]
 pub fn directed_complete_graph(
     py: Python,
     num_nodes: Option<usize>,
     weights: Option<Vec<PyObject>>,
     multigraph: bool,
 ) -> PyResult<digraph::PyDiGraph> {
-    directed_mesh_graph(py, num_nodes, weights, multigraph)
```

Comment:
Same applies for the directed version

Reply:
Done in https://github.com/Qiskit/rustworkx/pull/772/commits/0717ca68f6f99f2aa12a6328106ba9c142db3d31.